mono/metadata/w32file-unix.c

```diff
@@ -1400,7 +1402,7 @@ static guint32 file_seek(gpointer handle, gint32 movedistance,
 		return(INVALID_SET_FILE_POINTER);
 	}
 
-	mono_trace (G_LOG_LEVEL_DEBUG, MONO_TRACE_IO_LAYER, "%s: lseek returns %lld", __func__, newpos);
+	mono_trace (G_LOG_LEVEL_DEBUG, MONO_TRACE_IO_LAYER, "%s: lseek returns %lld", __func__, (long long) newpos);
```

Comment:
Please use `gint64` instead of `long long`, and use `guint64` instead of `long long unsigned`